sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -1201,6 +1249,30 @@ public TableReference getTable() {
       return toBuilder().setKmsKey(kmsKey).build();
     }
 
+    /**
+     * Sets the {@link Row} conversion function.
+     *
+     * <p>Both {@link #withToBeamRowFn(ToBeamRowFunction)} and {@link
+     * #withFromBeamRowFn(FromBeamRowFunction)} must be set to enable automatic Beam {@link Schema}
+     * support.
+     */
+    @Experimental(Experimental.Kind.SCHEMAS)
+    public TypedRead<T> withToBeamRowFn(ToBeamRowFunction<T> toRowFn) {
```

Comment:
Maybe merge WithToBeamRowFn and withFromBeamRowFn into a single method, as it never makes sense to call just one of them?